### DISCUSSION TITLE
Vindstød - Variabel Klima Spot

### DISCUSSION THREAD

**Discussion by KDYR** [rndfm/elspotpris#124] [Q&A]
Hi
I would like to help by adding tariff data for the Vindstød (new?) product named "Variabel Klima Spot" but couldn't see how you'd like to structure/code time-of-day dependent fees. Vindstød has informed that for this product the previous time-independent "Spottillæg" is replaced by a time-of-day dependent fee. Fee values for non-commercial consumers are:
00:00-05:00: 0 øre pr. kWh. 
05:00-09:00: 45 øre pr. kWh. 
09:00-13:00: 25 øre pr. kWh. 
13:00-22:00: 45 øre pr. kWh.
22:00-00:00: 25 øre pr. kWh.
"Certificate of Origin" fee is 29 kr per month

**Comment by rndfm** [comment]
That is one weird product.

An not supported by the platform as it is. I will look into this.

**Comment by rndfm** [comment]
Where can i find this product?

It's not mentioned at: https://www.vindstoed.dk/se-hvad-du-kan-spare-og-tilmeld

**Reply by KDYR** [reply]
I'll investigate with the company (Vindstød) to find an official product description. 
If you are curious, here is in the meantime a snip from the mail sent to me (I presume you read Danish):
```
**Stigning i spottillæg på dit elkøb og abonnement**
Vi ønsker at give dig sikkerhed for, at dit kWh-forbrug altid er bundet til elmarkedets pris, nemlig prisen på Nord Pool spot. Dette sker ved, at vi afregner dit forbrug til spotprisen time-for-time tillagt vores tillæg og et fast, månedligt abonnement.
Som følge af ændringerne i tarifmodellen samt Vindstøds ønske om at understøtte formålet med tarifmodellen, ændres dit spottillæg hos Vindstød til at være et bestemt tillæg i hver time. Tidligere havde du et ensartet tillæg i samtlige døgnets timer. Samtidig hæves dit abonnement. Disse justeringer er også en følge af de stigende elpriser i Europa.
Derfor indfører vi, at din maksimale elpris udregnes som vores faktiske omkostninger til:

- At indkøbe din el på børsen. Indkøbsprisen opgøres som den indkøbspris (øre pr. kWh som du forbruger), vi har i hver enkelt time på den nordiske elbørs (Nord Pool spot), med tillæg af vores faktiske balance- og handelsomkostninger til Energinet.dk samt bidrag til Vindstød. Tillægget til Vindstød udgør i timerne kl. 00:00-05:00: 0 øre pr. kWh. I timerne kl. 05:00-09:00: 45 øre pr. kWh. I timerne kl. 09:00-13:00: 25 øre pr. kWh. I timerne kl. 13:00-22:00: 45 øre pr. kWh. I timerne kl. 22:00-00:00: 25 øre pr. kWh. For erhverv udgør tillægget til Vindstød i timerne kl. 00:00-05:00: 10 øre pr. kWh. I øvrige timer er tillægget tilsvarende dem, som er angivet ovenfor for private. Priser er inkl. moms.

- At indkøbe oprindelsescertifikater fra vindmøller (”GoOs” dækkende det indkøbte elforbrug) med tillæg af et fast, månedligt abonnement på 29,00 kr. inkl. moms.

```

**Reply by KDYR** [reply]
Follow-up: Vindstød has responded to my mail without answering where one can see/obtain info about the product "Variabel Klima Spot". They also avoided to comment of the VAT issue.
They simply moved me to a product where I pay no monthly fee and only half a Danish cent pr kWh.
Vindstød may be withdrawing the product - not sure.
